Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -193,6 +193,26 @@ void WebFoundTextRangeController::didEndTextSearchOperation()
         m_webPage->corePage()->pageOverlayController().uninstallPageOverlay(*m_findPageOverlay, WebCore::PageOverlay::FadeMode::Fade);
 }
 
+void WebFoundTextRangeController::addFindOverlayLayer(CompletionHandler<void(WebCore::GraphicsLayer::PlatformLayerID)>&& completionHandler)
+{
+    if (!m_findPageOverlay) {
+        m_findPageOverlay = WebCore::PageOverlay::create(*this, WebCore::PageOverlay::OverlayType::Document);
+        m_webPage->corePage()->pageOverlayController().installPageOverlay(*m_findPageOverlay, WebCore::PageOverlay::FadeMode::DoNotFade);
+        m_findPageOverlay->layer().setOpacity(0);
+        m_findPageOverlay->layer().setAllowsTiling(false);
```

Comment:
What's this about! I would expect the find overlay to usually be tiled.

Reply:
(I see your comment in the commit message, but this seems like a mistake for large screen sizes?)

Reply:
Always-tiled is fine, we do that sometimes.